src/lwc/rd2PauseForm/rd2PauseForm.js

```diff
@@ -15,6 +15,7 @@ import selectedRowsSummarySingular from '@salesforce/label/c.RD2_PauseSelectedIn
 import saveSuccessMessage from '@salesforce/label/c.RD2_PauseSaveSuccessMessage';
 import deactivationSuccessMessage from '@salesforce/label/c.RD2_PauseDeactivationSuccessMessage';
 import rdClosedMessage from '@salesforce/label/c.RD2_PauseClosedRDErrorMessage';
+import elevateRdErrorMessage from '@salesforce/label/c.RD2_ElevateNotSupportErrorMessage';
```

Comment:
Let's rename `elevateRdErrorMessage` to `elevateNotSupported` or ``elevateNotSupportedMessage`